Open Judge System/Services/OJS.Services/Data/SubmissionsForProcessing/SubmissionsForProcessingDataService.cs

```diff
@@ -107,5 +134,33 @@ public void Update(SubmissionForProcessing submissionForProcessing)
             this.submissionsForProcessing.Update(submissionForProcessing);
             this.submissionsForProcessing.SaveChanges();
         }
+
+        private void AssignWorkerType(SubmissionForProcessing submissionForProcessing, Submission submission)
+        {
+            submissionForProcessing.WorkerType = submission.WorkerType;
+
+               if (submissionForProcessing.WorkerType == WorkerType.Default)
```

Comment:
```
if (submissionForProcessing.WorkerType != WorkerType.Default)
{
    return;
}
```

It would be better if the logic of the method is not in a conditional. Also unnesessary indentation there.

Reply:
Fixed